src/kOS/Suffixed/OrbitInfo.cs

```diff
@@ -30,6 +30,8 @@ private void InitializeSuffixes()
             AddSuffix("NAME", new Suffix<string>(() => name));
             AddSuffix("APOAPSIS", new Suffix<double>(() => orbit.ApA));
             AddSuffix("PERIAPSIS", new Suffix<double>(() => orbit.PeA));
+            AddSuffix("ETA_APOAPSIS", new Suffix<double>(() => orbit.timeToAp));
+            AddSuffix("ETA_PERIAPSIS", new Suffix<double>(() => orbit.timeToPe));
```

Comment:
On a vaguely related note, I've never liked how we have to have those weird exception cases for mapping ETA:Apoapsis to ETA_Apoapsis, and so on.  It parses like it was a suffix, but then the compiler goes out of its way to make a special case for it and change it into the underscore version, which also seems to have caused condition #662.

I dislike it because it's misleading to the script writers.  There is no such object as "ETA".  Therefore ETA:<anything> really shouldn't be a thing.

Unless... there really is an object called ETA so the exception case can be changed into something that is really using proper syntax. 

I'd like to propose that we either deprecate ETA:apoapsis and it's similar brethren, or if we want to keep it for compatibility, we actually make an ETA object so the parsing of it isn't a weird special case.  (i.e. An ETA object that holds the ETA for (or has runtime gettable properties for) apoapsis, periapsis, et all)